pkg/controllers/provisioningv2/rke2/bootstrap/controller.go

```diff
@@ -199,24 +191,11 @@ func (h *handler) assignPlanSecret(machine *capi.Machine, obj *rkev1.RKEBootstra
 		},
 	}
 
-	return []runtime.Object{sa, secret, role, rolebinding}, nil
-}
-
-func (h *handler) getMachine(obj *rkev1.RKEBootstrap) (*capi.Machine, error) {
-	for _, ref := range obj.OwnerReferences {
-		gvk := schema.FromAPIVersionAndKind(ref.APIVersion, ref.Kind)
-		if capi.GroupVersion.Group != gvk.Group ||
-			ref.Kind != "Machine" {
-			continue
-		}
-
-		return h.machineCache.Get(obj.Namespace, ref.Name)
-	}
-	return nil, generic.ErrSkip
+	return []runtime.Object{sa, secret, role, rolebinding}
 }
 
-func (h *handler) getEnvVar(machine *capi.Machine) (result []corev1.EnvVar, _ error) {
-	capiCluster, err := h.capiClusters.Get(machine.Namespace, machine.Spec.ClusterName)
+func (h *handler) getEnvVar(obj *rkev1.RKEBootstrap) (result []corev1.EnvVar, _ error) {
```

Comment:
very minor nit, but `obj` may be better declared with a more descriptive name for the common passerby

Reply:
I was just making it consistent with the rest of the functions in this file. They all call the bootstrap object `obj`.

I will update them all to be more descriptive.